ext/readline/tests/gh16812.phpt

```diff
@@ -0,0 +1,19 @@
+--TEST--
+GH-16812 readline_info(): UAF
+--EXTENSIONS--
+readline
+--SKIPIF--
+<?php
+if (READLINE_LIB != "libedit") die("skip libedit only");
+if(substr(PHP_OS, 0, 3) == 'WIN' ) {
+    die('skip not for windows');
+}
```

Comment:
Unless there is a particular reason to skip a test, I'd prefer to run it, even if the code path is not affected.